push_notifications/apns.py

```diff
@@ -239,18 +240,18 @@ def apns_send_bulk_message(registration_ids, alert, **kwargs):
 	it won't be included in the notification. You will need to pass None
 	to this for silent notifications.
 	"""
-	with closing(_apns_create_socket_to_push()) as socket:
+	with closing(_apns_create_socket_to_push(**kwargs)) as socket:
```

Comment:
**kwargs is already used in the _apns_send call.

Would be better to explicitly call out certfile and ca_certs anyway since they are well defined.